SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -131,12 +133,12 @@ To damage a player on a different server, the actor shooting the bullet must sen
 1. In your IDE, open `UnrealGDKThirdPersonShooter\Game\Source\ThirdPersonShooter\Characters\TPSCharacter.h`.
 1. On line 74, add this snippet:
 
-```
-UFUNCTION(CrossServer, Reliable)
-void TakeGunDamageCrossServer(float Damage, const struct FDamageEvent& DamageEvent, AController* EventInstigator, AActor* DamageCauser);
-```
+    ```
+    UFUNCTION(CrossServer, Reliable)
+    void TakeGunDamageCrossServer(float Damage, const struct FDamageEvent& DamageEvent, AController* EventInstigator, AActor* DamageCauser);
+    ```
 
-This snippet creates a new `UFUNCTION` marked with the function tags [CrossServer]({{urlRoot}}/content/cross-server-rpcs) and [Reliable (Unreal documentation)](https://wiki.unrealengine.com/Replication#Reliable_vs_Unreliable_Function_Call_Replication). The CrossServer tag forces this function to be executed as as cross-server RPC.
+    This snippet creates a new `UFUNCTION` marked with the function tags [CrossServer]({{urlRoot}}/content/cross-server-rpcs) and [Reliable (Unreal documentation)](https://wiki.unrealengine.com/Replication#Reliable_vs_Unreliable_Function_Call_Replication). The CrossServer tag forces this function to be executed as as cross-server RPC.
 
 3. In your IDE, open `UnrealGDKThirdPersonShooter\Game\Source\ThirdPersonShooter\Characters\TPSCharacter.cpp`.
 4. Replace the TakeDamage function (lines 514-548) with this snippet:
```

Comment:
```suggestion
1. Replace the TakeDamage function (lines 514-548) with this snippet:
```

Reply:
outdated, the slightly odd formatting is necessitated by improbadoc